templates/Test.cpp.ejs

```diff
@@ -93,7 +93,7 @@ void AutoGen<%- featureGroup.name %>Tests::ClassSetUp()
             assert(SUCCEEDED(hr));
             if (SUCCEEDED(hr))
             {
-                hr = PFGetAuthResult(&async, &entityHandle);
+                hr = PFAuthenticationClientLoginGetResult(&async, &entityHandle);
                 assert(SUCCEEDED(hr) && entityHandle != nullptr);
```

Comment:
is there a classic login method somewhere or is that next PR?

Reply:
Will be a future PR, haven't done that part yet